packages/istanbul-lib-instrument/src/instrumenter.js

```diff
@@ -80,7 +80,13 @@ class Instrumenter {
         const opts = this.opts;
         const ast = babylon.parse(code, {
             allowReturnOutsideFunction: opts.autoWrap,
-            sourceType: opts.esModules ? "module" : "script"
+            sourceType: opts.esModules ? "module" : "script",
+            plugins: [
+              'asyncGenerators',
+              'dynamicImport',
+              'flow',
+              'jsx'
+            ]
```

Comment:
Is there some way we can just pull this list in, like, is it exported from babylon or something?